src/components/common/Badge/Badge.tsx

```diff
@@ -0,0 +1,43 @@
+// The 'Badge' component.
+// This component accepts 3 arguments as props.
+// A mandatory Label (string)
+// A mandatory Variant ('default' | 'primary' | 'success' | 'error')
+// An optional icon (React.ReactNode)
+
+import BadgeInterface from "./BadgeInterface";
+
+const Badge = ({ label, icon, variant }: BadgeInterface) => {
+  const badgeTextVariants = {
+    default: "text-[#344054]",
+    primary: "text-[#B15210]",
+    success: "text-[#0B7C17]",
+    error: "text-[#A50909]",
+  };
+  const badgeBackgroundVariants = {
+    default: "bg-[#F2F4F7]",
+    primary: "bg-[#FFFAF7]",
+    success: "bg-[#E7F7E9]",
+    error: "bg-[#FDE7E7]",
+  };
```

Comment:
use the variables in the tailwind config (change the colors in the globals.css if need be)